packages/mongodb-rag-ingest/src/commands/all.test.ts

```diff
@@ -84,8 +78,8 @@ describe("allCommand", () => {
       expect(lastSuccessfulRunDate).toBeNull();
       await doAllCommand(
         {
-          pageStore: mockPageStore,
-          embeddedContentStore: mockEmbeddedContentStore,
+          pageStore: pageStore,
+          embeddedContentStore: embedStore,
```

Comment:
nit: i'd also update other variable names that'd easily support [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) (TIL the name for this)